api/core/v2/event.go

```diff
@@ -440,6 +442,13 @@ func EventFields(r Resource) map[string]string {
 	}
 }
 
+func isSilenced(e *Event) string {
+	if len(e.Check.Silenced) > 0 {
+		return "true"
+	}
+	return "false"
+}
+
```

Comment:
Shouldn't this function also consider whether or not its entity is silenced?

Reply:
Long story short, we already do but we store the result in `Check` (I have no idea why though): https://github.com/sensu/sensu-go/blob/69c0ef22f3d7d790a3a341d5b7d43489113c027c/backend/eventd/silenced.go#L23-L39